lib/krane/deploy_task.rb

```diff
@@ -241,7 +241,7 @@ def discover_resources
       crds_by_group_kind = cluster_resource_discoverer.crds.group_by(&:group_kind)
       @template_sets.with_resource_definitions(current_sha: @current_sha, bindings: @bindings) do |r_def|
         group, kind = group_kind_for_r_def(r_def)
-        crd = crds_by_group_kind[group + "/" + kind]&.first
+        crd = crds_by_group_kind["#{kind}.#{group}"]&.first
```

Comment:
I've adopted the convention of displaying group-kind information as `$KIND.$GROUP`, which mirrors how tools like kubectl, etc. work. e.g. `deployments.apps`